kettle/Makefile

```diff
@@ -44,5 +44,21 @@ update-staging: get-cluster-credentials
 rollback: get-cluster-credentials
 	kubectl rollout undo deployments/kettle
 
+ensure-requirements3:
+	../hack/run-in-python-container.sh pip3 install -r requirements3.txt
```

Comment:
is this cheap to do repeatedly or should we follow-up with making it depend on the requirements3.txt file / outputting a dummy marker so it can be skipped when requirements are unchanged?

Reply:
yeah it's pretty cheap, and for this case makefile is smart enough to only do it once

Reply:
well, once *per `make` invocation*, but every invocation will do this again?

Reply:
yeah that's true, and I would say it's very fast once all requirements were met